lighting_reporting_vanguard_product/report/report_vanguard.py

```diff
@@ -12,6 +12,7 @@ class ProductReport(models.AbstractModel):
     def _get_report_values(self, docids, data=None):
         model = "lighting.product"
         lang = self.env.lang
+        company = self.env.company
```

Comment:
This and all the other changes should be done on Classic as well. During test phase we're gonna need Classic to work in production and it cannot fail at all. No only Vanguard but Classic should also be tested intensively.